### ISSUE TITLE
threat.enrichments should be normalized to an array

### ISSUE THREAD

**Issue by rmariko** [elastic/ecs#1607]
**Description of the issue:**
The [threat.enrichments field](https://www.elastic.co/guide/en/ecs/current/ecs-threat.html#field-threat-enrichments) is specified in the description and [this example](https://github.com/elastic/ecs/blob/master/rfcs/text/0021-threat-enrichment.md#example-document) to be an array of objects. It is specified in the threat schema ([threat.yml](https://github.com/elastic/ecs/blob/master/schemas/threat.yml#L17-L24)) to be of `type: nested` but I would have expected the `normalize` field to also be specified.

**Current:**
```
   - name: enrichments
      level: extended
      type: nested
      short: List of objects containing indicators enriching the event.
      beta: This field is beta and subject to change.
      description: >
        A list of associated indicators objects enriching the event, and the context of
        that association/enrichment.
```

**Expected:**
```
    - name: enrichments
      level: custom
      type: nested
      short: List of objects containing indicators enriching the event.
      beta: This field is beta and subject to change.
      description: >
        A list of associated indicators objects enriching the event, and the context of
        that association/enrichment.
      normalize:
        - array
```

**Comment by kgeller** [comment]
Thanks for the discovery @rmariko ! The fix is in and will be released with ECS 1.12.0

**Comment by rmariko** [comment]
Awesome, thanks!
